lib/components/Profile.tsx

```diff
@@ -24,6 +24,7 @@ const SocialIcon = ({ type, url, size = 24 }: SocialIconProps) => {
       icon = <Mail size={size}/>
       break
     case 'linkedin':
+      // TODO find an alternative icon
```

Comment:
For LinkedIn? Why exactly?

Reply:
Because Brand Icons for GitHub and LinkedIn are deprecated in lucide-react